cogs/cog_timer.py

```diff
@@ -29,29 +48,35 @@ async def set_timer(self, ctx, time: str, *msg: str):
             description="Timer created!\n",
             colour=cfg.colors.SUCCESS
         ))
-        # await async_tasks.handle_timers()
 
     @commands.command(name="list-timers", aliases=['lt'])
     async def list_timers(self, ctx, limit=None):
+        """
+        Lists all active timers created by the user.
+        Note: +limit -> earliest timers, -limit -> latest timers
+        """
 
-        # Identify users and fetch their timers
-        userid = ctx.author.id
+        # Identify user and fetch their timers
+        user = ctx.author
         user_timers = sorted(
-            [obj for obj in timer_priority_queue.TimerPriorityQueue.get_instance().user_map[userid] if type(obj) == timer.Timer]
+            [obj for obj in timer_priority_queue.TimerPriorityQueue.get_instance().user_map.get(user.id, []) if
+             type(obj) == timer.Timer]
```

Comment:
I can see why you used .get here, but you could check user_map earlier to ensure the list isn't empty